src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml

```diff
@@ -7,14 +7,33 @@
 }
 @if (!String.IsNullOrWhiteSpace(publicKey)) {
```

Comment:
I believe this was supposed to be changed to @if (HasText(publicKey)) as well and thought I did but it appears to have not saved.